index.js

```diff
@@ -418,6 +418,13 @@ class HyperBee {
     return iteratorToStream(new DiffIterator(new Batch(this, null, false, opts), new Batch(right, null, false, opts), opts), active)
   }
 
+  async hasPrefix (prefix) {
+    const ite = this.createRangeIterator({ gte: prefix })
```

Comment:
@rafapaezbas add limit: 1 here also - that'll make it a bit faster

Reply:
checkPrefix?